app/src/main/java/com/android/wardrobeManager/ui/images/PolkaDots.java

```diff
@@ -0,0 +1,81 @@
+package com.android.wardrobeManager.ui.images;
+
+import android.graphics.Bitmap;
+import java.util.Map;
+
+public class PolkaDots implements DesignFilterManager.DesignFilter {
+
+    protected Map<String, DesignFilterManager.DesignFilter> filters;
+    private int spacing = 80;
+    private int dotRadius = 10;
+    private int colorIndex = 0;
+
+    public Bitmap filter(Bitmap bitmap, Bitmap ref, int[] colors) {
+
+
+        for (int i = 0; i < bitmap.getWidth() * bitmap.getHeight(); i++) {
+
+            int x = i % bitmap.getWidth();
+            int y = i / bitmap.getWidth();
+
+            if (colors.length < 2) {
+                if (ref.getPixel(x, y) != 0xFFFFFFFF) {
+                    bitmap.setPixel(x, y, 0xFFEEEEEE);
+                }
+            } else {
+                if (ref.getPixel(x, y) != 0xFFFFFFFF) {
+                    bitmap.setPixel(x, y, colors[0]);
+                }
+            }
+
+
+        }
+
```

Comment:
Get rid of extra spacing here, in general you should never have more than a single blank line between code blocks. Unless it is temporary for some reason.